### PR TITLE
Update sit-get-started-exact-data-match-based-sits-overview.md

### PR DESCRIPTION
**Pull request by msbemba**
Added license information

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9460

File: microsoft-365/compliance/sit-get-started-exact-data-match-based-sits-overview.md [hunk 1]
```diff
@@ -76,7 +76,12 @@ You can find out what region your tenant is hosting data-at-rest in by following
 
 You must be a global admin, compliance administrator, or Exchange Online administrator to perform the tasks described in this article. To learn more about DLP permissions, see [Permissions](data-loss-prevention-policies.md#permissions).
 
-See the [data loss prevention service description](/office365/servicedescriptions/microsoft-365-service-descriptions/microsoft-365-tenantlevel-services-licensing-guidance/microsoft-365-security-compliance-licensing-guidance#data-loss-prevention-for-exchange-online-sharepoint-online-and-onedrive-for-business) for complete licensing information
+You must have one of the following licenes,
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sit-get-started-exact-data-match-based-sits-overview.md]:
```suggestion
You must have one of the following licenses:
```

File: microsoft-365/compliance/sit-get-started-exact-data-match-based-sits-overview.md [hunk 2]
```diff
@@ -76,7 +76,12 @@ You can find out what region your tenant is hosting data-at-rest in by following
...
+
+Office 365 E5/A5/G5
+Microsoft 365 E5/A5/G5
+Microsoft 365 E5/A5/F5/G5 Compliance and Microsoft 365 F5 Security & Compliance
+Microsoft 365 E5/A5/F5/G5 Information Protection and Governance
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sit-get-started-exact-data-match-based-sits-overview.md]:
```suggestion
- Office 365 E5/A5/G5
- Microsoft 365 E5/A5/G5
- Microsoft 365 E5/A5/F5/G5 Compliance and Microsoft 365 F5 Security & Compliance
- Microsoft 365 E5/A5/F5/G5 Information Protection and Governance
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks

**Comment by chrfox** [comment]
closing this without merging, the link to the service descriptions is the best way to present this

**Pull request closed without merging**